docs/tutorial/03-fungible-tokens.mdx

```diff
@@ -228,90 +271,34 @@ self.balance = self.balance - amount
 ```
 
 In Solidity, this could be a risk for integer overflow or underflow, but Cadence has built-in overflow and underflow protection, so it is not a risk.
-We are also using unsigned integers in this example, so the vault`s balance cannot go below 0.
+We are also using unsigned numbers in this example, so the vault`s balance cannot go below 0.
```

Comment:
We mention this advantage above, so if we wish to re-mention it we could say "As we mentioned above...".